config/esw.yaml

```diff
@@ -0,0 +1,8 @@
+gps_driver:
+  port: "/dev/gps"
+  baud: 115200
+
+imu_driver:
+  port: "/dev/imu"
```

Comment:
nice

Reply:
Oh as a random note, is the udev rule something that we will configure with ros or is it something that people who set up the jetson need to remember? I feel like there should be a "Setting up a new Jetson" guide somewhere in the wiki if its the latter. Otherwise, maybe we can do a script similar to how Riley made the networking configuration script on my branch

Reply:
+1 on the wiki page.

Also depending on how many times we plan to brick Jetsons this year a Jetson-specific variant of the install script with automatic configuration for udev/network/etc. might prove to be useful.